src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -1942,8 +2054,17 @@ private List<Document> generatePackA(CaseData caseData, List<Document> staticDoc
     }
 
     private List<Document> getWitnessStatement(CaseData caseData) {
-        return !CollectionUtils.isEmpty(caseData.getFl401UploadWitnessDocuments()) ? ElementUtils.unwrapElements(
-            caseData.getFl401UploadWitnessDocuments()) : new ArrayList<>();
+        List<Document> witnessStatements = new ArrayList<>();
+        if (!CollectionUtils.isEmpty(caseData.getFl401UploadWitnessDocuments())) {
+            witnessStatements.addAll(ElementUtils.unwrapElements(
+                caseData.getFl401UploadWitnessDocuments()));
+        }
+        if (!CollectionUtils.isEmpty(caseData.getCourtNavUploadedDocs())) {
```

Comment:
Should be checking for document type for witness statements from courtnav uploaded docs